db/service_updates.yml

```diff
@@ -0,0 +1,5 @@
+- id: 1
+  date: '2021-09-01'
+  title: Lead and employing schools
```

Comment:
We've tweaked the content for this slightly:

title: You no longer need to provide a lead or employing school for trainees who are employed or privately funded

Content: >-
When you’re registering a trainee, we were asking for a lead or employing school in all cases. If a trainee is employed or funded privately we do not need this information so we’ve added a ‘not applicable’ option to the lead and employing schools sections. You do not need to provide lead or employing schools if the trainee is employed or funded privately.